howto/setup-pub-sub-message-broker/setup-nats-streaming.md

```diff
@@ -0,0 +1,91 @@
+# Setup NATS
+
+## Locally
+
+You can run a NATS server locally using Docker:
+
+```bash
+docker run -d -name nats-streaming -p 4222:4222 -p 8222:8222 nats-streaming
+```
+
+You can then interact with the server using the client port: `localhost:4222`.
+
+## Kubernetes
+
+Install NATS on Kubernetes by using the [kubectl](https://docs.nats.io/nats-on-kubernetes/minimal-setup):
+
+```bash
+# Single server NATS
+
+kubectl apply -f https://raw.githubusercontent.com/nats-io/k8s/master/nats-server/single-server-nats.yml
```

Comment:
This line should be removed, no?

Reply:
no, if install NATS streaming this way. should install Nats and NATS streaming both. Because NATS streaming use Nats.